### PR TITLE
Update home page

### PR DESCRIPTION
**Pull request by alflennik**
Updates the home page with new copy and an embedded copy of our brand new introductory video! The video is currently a placeholder but will be replaced with the live video later on the vimeo side.

### CODE REVIEW

**Comment by s3ththompson** [comment]
@alflennik looks great, just a few nits:

1. We should add in:

    > [Read more](https://github.com/w3c/aria-at/wiki/How-Gaps-in-Assistive-Technology-Interoperability-Hinder-Inclusion) about how the AT interoperability gap hinders inclusion on the web for people with disabilities"
    
    under the last paragraph in the first section.

1. The Scalable Automated Testing card should have [the icon with stacks and a square ruler](https://aria-at.w3.org/b94a228c191bd738b564128bf6c5b467.jpg)

1. Under Scalable Automated Testing, "explore the code repository" should link to https://github.com/bocoup/at-automation-experiment.

**Comment by alflennik** [comment]
@s3ththompson Thank you for the careful check! I've fixed those three issues and deployed to the [sandbox](https://aria-at-app-sandbox.bocoup.com/) - take a look and let me know if you notice anything else. The makes the one outstanding issue the lack of a live video link or placeholder.

**Comment by alflennik** [comment]
@howard-e I was hoping you could review the code Howard! The PR is WIP until we have the video link but the code itself should be production ready.

**Review by howard-e — APPROVED** [review]
LGTM

File: client/components/Home/Home.jsx [hunk 1]
```diff
@@ -1,180 +1,170 @@
...
+                                reader behaviors for 40 examples of common web
+                                design patterns. View the{' '}
+                                <a href="https://github.com/w3c/aria-at/blob/master/metrics/coverage.md">
+                                    Test Writing Progress Report
+                                </a>{' '}
+                                and{' '}
+                                <a href="https://w3c.github.io/aria-at/build/">
+                                    view draft test plans
+                                </a>
+                                .
```
Comment (howard-e) [inline client/components/Home/Home.jsx:86]: Following internal discussions and confirmation, this period may cause unexpected behaviors in the screen reader output when immediately following the link. I can consistently reproduce an issue here in VoiceOver on Chrome while in reading mode.

File: client/components/Home/Home.jsx [hunk 2]
```diff
@@ -1,180 +1,170 @@
...
-                            <Link to="/reports">Review test results</Link>
+                            <img src={iconJoinCommunity} alt="" />
+                            <h3>Building industry consensus</h3>
+                            <p>
+                                Once a pattern has a reviewed test plan with
+                                results data, a candidate report is published
+                                and the process of building consensus around the
+                                plan begins. View reports generated from
+                                candidate test plans on the{' '}
+                                <Link to="/reports">reports page</Link>.
```
Comment (howard-e) [inline client/components/Home/Home.jsx:108]: Same comment as before

**Review by howard-e — CHANGES_REQUESTED** [review]

File: client/components/Home/Home.jsx (hunk 1 above)
Comment (s3ththompson) [reply client/components/Home/Home.jsx:86]: Let's change this to (link in bold):

> view the **draft test plans** preview.

File: client/components/Home/Home.jsx (hunk 2 above)
Comment (s3ththompson) [reply client/components/Home/Home.jsx:108]: Let's change this to (link in bold):

> on the **reports** page.

File: client/components/Home/Home.jsx [hunk 3]
```diff
@@ -1,180 +1,170 @@
...
-                            <a href="https://w3c.github.io/aria-at/">
-                                Review existing test plans
-                            </a>
+                            <img src={iconReviewTests} alt="" />
+                            <h3>Testing proposed expectations</h3>
+                            <p>
+                                This website enables us to manage test data, run
+                                tests with multiple testers, review results, and
+                                publish reports. View our progress on the{' '}
+                                <Link to="/test-queue">test queue page.</Link>
```
Comment (s3ththompson) [inline client/components/Home/Home.jsx:96]: Let's change this to (link in bold):

> on the **test queue** page.

File: client/components/Home/Home.jsx [hunk 4]
```diff
@@ -1,180 +1,170 @@
...
-                        </Link>
+                                Read more about how the AT interoperability gap
+                                hinders inclusion on the web for people with
+                                disabilities.
+                            </a>
+                        </p>
+                    </div>
+                    <div className="hero-video">
+                        <iframe
+                            src="https://player.vimeo.com/video/651279608?h=45aefd646f"
```
Comment (s3ththompson) [inline client/components/Home/Home.jsx:56]: Should be `https://player.vimeo.com/video/651279608?h=45aefd646f&byline=false&dnt=true&portrait=false` to hide byline and portrait (please verify on your end, I haven't been able to preview these query params

Comment (alflennik) [reply client/components/Home/Home.jsx]: Verified. Video looks awesome can't wait to watch the whole thing later 🤓

File: client/components/Home/Home.jsx (hunk 3 above)
Comment (alflennik) [reply client/components/Home/Home.jsx]: Fixed!

File: client/components/Home/Home.jsx (hunk 2 above)
Comment (alflennik) [reply client/components/Home/Home.jsx]: Fixed!

File: client/components/Home/Home.jsx (hunk 1 above)
Comment (alflennik) [reply client/components/Home/Home.jsx]: Fixed!

**Review by howard-e — APPROVED** [review]
There still seems to be an unexpected behavior I can consistently reproduce with the _MacOS 12.0.1 on Chrome Version 96.0.4664.55 (Official Build) (x86_64)_ combination. Approving in the interest of the time related constraint of this PR.

**Pull request merged**